docs/tac.md

```diff
@@ -0,0 +1,60 @@
+## Setup
+
+Clone the repo to include submodules.
+
+``` bash
+git clone git@github.com:fetchai/agents-tac.git --recursive && cd agents-tac
+```
+
+
+Create and launch a virtual environment.
+
+``` bash
+pipenv --python 3.7 && pipenv shell
+```
+
```

Comment:
A step is missing here, Diarmid has merged into develop on tac now. Please have a look at the updated main readme there.

Reply:
Included.